format/spec.md

```diff
@@ -1766,4 +1766,4 @@ The Geometry and Geography class hierarchy and its Well-known text (WKT) and Wel
 
 Points are always defined by the coordinates X, Y, Z (optional), and M (optional), in this order. X is the longitude/easting, Y is the latitude/northing, and Z is usually the height, or elevation. M is a fourth optional dimension, for example a linear reference value (e.g., highway milepost value), a timestamp, or some other value as defined by the CRS.
 
-The version of the OGC standard first used here is 1.2.1, but future versions may also used if the WKB representation remains wire-compatible.
\ No newline at end of file
+The version of the OGC standard first used here is 1.2.1, but future versions may also used if the WKB representation remains wire-compatible.
```

Comment:
What happened here?

Reply:
Ugh my editor keeps doing endline changes here. I'll redo the change in vim

Reply:
Ok looks like there wasn't a newline here before. I may have accidentally removed it when I fixed a typo here within the github editor in a different commit.